apollo-training/src/modules/trainee/mutation.js

```diff
@@ -1,16 +1,24 @@
+import pubsub from '../pubsub';
+import constant from '../../lib/constant';
 import userInstance from '../../service/user';
 
 export default {
   createTrainee: (parent, args) => {
     const { user } = args;
-    return userInstance.createUsers(user);
+    const addedUser = userInstance.createUsers(user);
+    pubsub.publish(constant.subscriptions.TRAINEE_ADDED, { traineeAdded: addedUser });
```

Comment:
What publish method does?

Reply:
Its concept coming from socket.io like instant notification, to publish over all subscription(to instant notify).